optimism/src/keccak/witness.rs

```diff
@@ -61,11 +62,18 @@ impl<F: Field> Default for Env<F> {
     }
 }
 
-impl<F: Field> KeccakInterpreter<F> for Env<F> {
+impl<F: Field> ArithHelpers<F> for Env<F> {
+    fn two_pow(x: u64) -> <Env<F> as Interpreter<F>>::Variable {
+        Self::constant_field(F::two_pow(x))
+    }
+}
+
+impl<F: Field> BoolHelpers<F> for Env<F> {}
```

Comment:
Yes, so this seems weird. If these functions were declared on the top level, one wouldn't need these declarations. But maybe it's easier like this. Dunno.